src/components/UploadVideo/UploadVideo.tsx

```diff
@@ -40,8 +40,8 @@ const UploadVideo = () => {
       const {filesArray, newFramesArray, videoId} = transformRawFrameData(rawFrameDataArray); // TODO refactor so we can access images from the dashboard (useContext?) and trigger the request to be, s3 and be again one after another.
       setFramesArray(newFramesArray);
       const urls: string[] = sendDataToBackEnd(newFramesArray, videoId); // TODO it's gonna be async
-      const isUploaded = await uploadImgToBucket(filesArray, urls); // make it return a boolean if responses % is high enough
-      const analytics = getAnalytics(videoId); // TODO pass analytics to helper functions and then to dashboard
+      const isUploaded = await uploadImgToBucket(filesArray, urls);
+      isUploaded && getAnalytics(videoId); // TODO pass analytics to helper functions and then to dashboard atm we're logging a string
       setAlertMessage(uploadSuccessful);
```

Comment:
getAnalytics gets the data of the video analysis? Maybe call it Analysis, Analytics made me think that you were getting Analytics from the speed of the upload or something

Reply:
ahahah alright, I'll change that